test-butler-library/src/main/java/com/linkedin/android/testbutler/TestButler.java

```diff
@@ -262,6 +262,21 @@ public static void setSpellCheckerEnabled(boolean enabled) {
     }
 
     /**
+     * Enable or disable the system software keyboard
```

Comment:
Let's make this similar to the docs on `ShowImeWithHardKeyboardHelper`, this isn't really accurate